### ISSUE TITLE
[Bug]: some ut failed

### ISSUE THREAD

**Issue by zhangxu19830126** [matrixorigin/matrixone#11335]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

ut pass

### Expected Behavior

ut failed. 
1. TestAll
[logs_166508.zip](https://github.com/matrixorigin/matrixone/files/12385966/logs_166508.zip)

https://github.com/matrixorigin/matrixone/actions/runs/5911306556/job/16033718751?pr=11312

**Comment by XuPeng-SH** [comment]
next release

**Comment by XuPeng-SH** [comment]
assign to @jiangxinmeng1

**Comment by jiangxinmeng1** [comment]
fixed by #11575
